app/collections/create-form.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import apiClient from 'panoptes-client/lib/api-client';
+
+class CollectionsCreateForm extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      collectionNameLength: 0,
+      error: null
+    };
+
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleNameInputChange = this.handleNameInputChange.bind(this);
+  }
+
+  onSubmit(event) {
+    event.preventDefault();
+
+    const displayName = this.name.value;
+    const isPrivate = this.isPrivate.checked;
+
+    const links = {};
+
+    if (this.props.projectID) {
+      links.project = this.props.projectID;
+    }
+
+    if (this.props.subjectIDs.length > 0) {
+      links.subjects = this.props.subjectIDs;
+    }
+
+    const collection = {
+      display_name: displayName,
+      private: isPrivate,
+      links
+    };
+
+    apiClient.type('collections').create(collection).save()
+      .then((newCollection) => {
+        this.name.value = '';
+        this.isPrivate.value = true;
```

Comment:
This is non-blocking because it was written this way before, but I don't understand why this is being set true after a successful collection POST. This is something to consider for future refactoring when the description field is added.

Reply:
Agreed, I was wondering the same thing. Happy to change to whatever makes the most sense - `false`, `null`, or `undefined`, just let me know what you'd prefer :)